modules/auth/repo_form.go

```diff
@@ -220,6 +220,16 @@ func (f *CreateLabelForm) Validate(ctx *macaron.Context, errs binding.Errors) bi
 	return validate(errs, ctx.Data, f, ctx.Locale)
 }
 
+// Label templates
+
+type InitializeLabelsForm struct {
+	TemplateName string `form:"templatename" binding:"Required"`
```

Comment:
I dont know why, but without `form:"templatename"` I can't retrieve value from HTML form and I get the error `TemplateName cannot be empty.`

![image](https://cloud.githubusercontent.com/assets/1005086/18033653/f9dcb79c-6d2a-11e6-89c1-b86eda8d0e2c.png)